react-intl/index.d.ts

```diff
@@ -227,7 +228,11 @@ declare namespace ReactIntl {
             defaultFormats?: Object;
         }
     }
-    class IntlProvider extends React.Component<IntlProvider.Props, any> { }
+    class IntlProvider extends React.Component<IntlProvider.Props, any> {
+        getChildContext(): {
+            intl: any;
```

Comment:
What type it should be here? Should it be `InjectedIntl` here? Can you please add it?